src/screens/TopicPageScreen/index.js

```diff
@@ -337,10 +348,9 @@ const TopicPageScreen = (props) => {
     return () => {
       if (interactionManagerRef.current) interactionManagerRef.current.cancel();
       if (interactionManagerAnimatedRef.current) interactionManagerAnimatedRef.current.cancel();
-
       unsubscribe();
     };
-  }, [navigation]);
+  }, [navigation, coverPath]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The cleanup function of this `useEffect` hook now also depends on `coverPath`. This ensures that any ongoing animations are cancelled when the component unmounts or `coverPath` changes.


<!-- This is an auto-generated comment by OSS CodeRabbit -->